imports/api/playlists/playlist_index.js

```diff
@@ -0,0 +1,21 @@
+import { Playlists } from './playlists_collection.js';
+import { Index, MinimongoEngine } from 'meteor/easy:search';
+import { _ } from 'meteor/underscore';
+
+export const PlaylistsIndex = new Index({
+  engine: new MinimongoEngine({
+    sort: function() {
+      return { showDate: -1 };
+    },
+    selector: function (searchObject, options, aggregation) {
+      let selector = this.defaultConfiguration().selector(searchObject, options, aggregation);
+      return selector;
+    }
+  }),
+  collection: Playlists,
+  fields: ['showId'],
+  defaultSearchOptions: {
+    limit: 20
+  },
+  permission: () => true
+});
```

Comment:
Please remove any unused code from the repository. We want to avoid bloat as much as possible.